docs/topic_guides/evaluation/clusters/how_to_compute_metrics.md

```diff
@@ -0,0 +1,42 @@
+# How to compute graph metrics with Splink
+
+To enable users to calculate a variety of graph metrics for their linked data, Splink provides the `compute_graph_metrics()` method.
+
+The method is called on the `linker` like so:
+
+```
+linker.computer_graph_metrics(df_predict, df_clustered, threshold_match_probability=0.95)
```

Comment:
Maybe we could even include a full small example that has every step up to this one. So starting with a simple model, and ending with computing the metrics + looking at the frame, for people to copy/paste and try out

Reply:
like
```py
...  # imports, settings
linker = DuckDBLinker(df, settings)
... # train
df_predict = linker.predict()
df_clustered = linker.cluster_pairwise_predictions_at_threshold(df_predict, 0.95)
graph_metrics = linker.computer_graph_metrics(df_predict, df_clustered)

graph_metrics.clusters.as_pandas_dataframe()
```